api/auth/user_management.py

```
@@ -291,6 +291,39 @@ async def wrapper(request: Request, *args, **kwargs):
+                # Authenticated - set user info
+                email = user_info.get("email")
+                request.state.user_id = base64.b64encode(email.encode()).decode()
+                request.state.user_email = email
+            else:
+                # Not authenticated - set to None (allow demo mode)
+                request.state.user_id = None
+                request.state.user_email = None
+
+            return await func(request, *args, **kwargs)
+
+        except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)